src/v/net/server.cc

```diff
@@ -324,6 +325,31 @@ void server::setup_metrics() {
          sm::description(ssx::sformat("{}: Latency ", cfg.name)))});
 }
 
+void server::setup_public_metrics() {
+    namespace sm = ss::metrics;
+    if (!_proto) {
+        return;
+    }
+
+    auto aggregate_labels = config::shard_local_cfg().aggregate_metrics()
+                              ? std::vector<sm::label>{sm::shard_label}
+                              : std::vector<sm::label>{};
```

Comment:
The aggregation should be always on for `/public_metrics`

Reply:
Done.